src/SIM/SIM2D.C

```diff
@@ -78,6 +78,10 @@ bool SIM2D::addConnection (int master, int slave, int mIdx,
     return false;
   }
 
+  adm.dd.ghostConnections.insert(DomainDecomposition::Interface{master, slave,
+                                                                mIdx, sIdx, orient,
+                                                                dim, basis, thick});
+
```

Comment:
Intet problem, prøver bare å forstå litt. Trodde at adm og dd var for å støtte paralell løsning, men nå er det altså ting her som også trengs seriell multi-patch. Så kan hende det er mer logisk at ghostConnections ligger for seg selv?